packages/web/public/static/locales/it/authentication.json

```diff
@@ -1,56 +1,56 @@
+    "submitButtonText": "Ripristina la password",
+    "successMessage": "La tua password è stata ripristinata con successo! Prova ad accedere",
+    "invalidOldPasswordError": "La vecchia password non è corretta",
+    "passwordsDoNotMatchError": "Le password non corrispondono",
+    "invalidResetTokenError": "Questo token di reset è invalido o scaduto"
   },
-  "handleInputLabel": "Handle",
-  "handleInputHelperMessage": "Can be letters, numbers, -, _, and cannot contain spaces",
-  "handleInputPlaceholder": "Your name, or perhaps a fun pseudonym!",
-  "handleRequiredErrorMessage": "Handle is required",
-  "handleValidationErrorMessage": "Handle may only contain letters, numbers, _, -, with no spaces",
-  "handleMinimumErrorMessage": "Your handle must be at least 3 characters",
+  "handleInputLabel": "Nome utente",
+  "handleInputHelperMessage": "Può contenere lettere, numeri, - e _, ma non spazi",
+  "handleInputPlaceholder": "Il tuo nome, o magari uno pseudonimo simpatico!",
+  "handleRequiredErrorMessage": "'Nome utente' è un campo obbligatorio",
+  "handleValidationErrorMessage": "Il nome utente può contenere solo lettere, numeri, _, -, ma nessuno spazio",
+  "handleMinimumErrorMessage": "Il tuo nome utente deve avere almeno tre caratteri",
   "emailInputLabel": "Email",
-  "emailInputPlaceholder": "Your email address",
-  "emailRequiredErrorMessage": "Email is required",
-  "emailValidationErrorMessage": "Invalid email address",
+  "emailInputPlaceholder": "Il tuo indirizzo email",
+  "emailRequiredErrorMessage": "'Email' è un campo obbligatorio",
+  "emailValidationErrorMessage": "Indirizzo email non valido",
   "passwordInputLabel": "Password",
-  "passwordInputPlaceholder": "A secure password",
-  "passwordRequiredErrorMessage": "Password is required",
-  "passwordValidationErrorMessage": "Invalid password",
-  "passwordMinimumErrorMessage": "Password must be at least 6 characters",
-  "confirmPasswordInputLabel": "Confirm Password",
-  "confirmPasswordInputPlaceholder": "Re-enter your new password",
-  "confirmPasswordRequiredErrorMessage": "Confirm password is required",
-  "confirmPasswordValidationErrorMessage": "Invalid confirmPassword address",
-  "goToSignupText": "Don't have an account?",
-  "goToSignupLink": "Sign up",
-  "goToRequestResetText": "Having trouble logging in?",
-  "goToRequestResetLink": "Reset your password"
+  "passwordInputPlaceholder": "Una password sicura",
+  "passwordRequiredErrorMessage": "'Password' è un campo obbligatorio",
+  "passwordValidationErrorMessage": "Password non valida",
+  "passwordMinimumErrorMessage": "La tua password deve avere almeno sei caratteri",
+  "confirmPasswordInputLabel": "Conferma password",
+  "confirmPasswordInputPlaceholder": "Reinserisci la tua nuova password",
+  "confirmPasswordRequiredErrorMessage": "'Conferma password' è un campo obbligatorio",
+  "confirmPasswordValidationErrorMessage": "Indirizzo confirmPassword non valido", // what is it?
```

Comment:
Hmm this string seems to have gotten messed up at some point, it should be something like "Your passwords do not match" (used when someone is updating their password and we ask them to confirm the password)

Reply:
Makes sense!